lib/travis/build/addons/jwt.rb

```diff
@@ -0,0 +1,43 @@
+require 'travis/build/addons/base'
+require 'jwt'
+
+module Travis
+  module Build
+    class Addons
+      class Jwt < Base
+        SUPER_USER_SAFE = true
+
+        def before_before_script
+          secrets = nil
+          if config.is_a?(String)
+            secrets = [config]
+          elsif config.is_a?(Array)
+            secrets = config
+          else
+            secrets = config.values
+          end
+
+          tokens = {}
+          secrets.each do |secret|
+            key, secret = secret.split('=').map(&:strip)
+            pull_request = self.data.pull_request ? self.data.pull_request : ""
+            now = Time.now.to_i()
+            payload = {"iss" => "travis-ci.org",
```

Comment:
Not sure if this is a reasonable issuer. Keep in mind that it could come from other domains, more generic value, say `Travis CI, GmbH` might be better.

Also, it is worth noting this value (whatever we end up using) in the document so that other third-party implementer could verify `iss`.